src/services/auth.service.ts

```diff
@@ -1,8 +1,8 @@
+import { Http, Headers, URLSearchParams, RequestOptions } from '@angular/http';
```

Comment:
@DamengRandom this line is no longer required

Reply:
Yup sure @trtshen